[OSDOCS#17176]: Make changes as per the errors and warnings for CQA-2 (#114253)
openshift/openshift-docs: security/container_security/security-network.adoc

toc::[]

[role="_abstract"]

> Network security can be managed at several levels. At the pod level, network namespaces can prevent containers from seeing other pods or the host system by restricting network access. Network policies give you control over allowing and rejecting connections. You can manage ingress and egress traffic to and from your containerized applications.

Reviewer: Too long
```suggestion
You can manage network security at several levels, such as by using network namespaces and network policies.

At the pod level, network namespaces can prevent containers from seeing other pods or the host system by restricting network access. Network policies give you control over allowing and rejecting connections. You can manage ingress and egress traffic to and from your containerized applications.
```
